.github/workflows/build.yml

```diff
@@ -11,6 +11,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: save check run id
+        run: |
+          CHECK_SUITE_URL=$(curl -s -H "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID} | jq -r '.check_suite_url')
+          CHECK_RUN_ID=$(curl -s -H "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -H "Accept: application/vnd.github.antiope-preview+json" $CHECK_SUITE_URL/check-runs | jq '.check_runs[] | select(.name=="build-and-verify") | .id ')
+          mkdir -p ./pr
+          echo $CHECK_RUN_ID > ./pr/run-id
```

Comment:
where is `run-id` used, i.e. read?

Reply:
I have sent another PR #77 
It's used there to link to the error log console.
https://github.com/openshift-helm-charts/repo/pull/77/files#diff-e29b824e0807b8c6fe5c1e8538353b8a0dd88850281bce900dfb90bd9f73520dR6-R11

Reply:
If it used there, then please remove it from this one and move it overthere.

Reply:
This job doesn't have access to create comments. So, all the required information is captured in the PR artifact. Later the other job create comments based on the PR artifact.

Reply:
please keep this PR focused., introduced step "save check run id" is not consumed within this PR. Hence, if it is consumed in #77 - then add "save check run id" within that PR.

Reply:
This PR is targeted at `dev` branch whereas the other PR is targeted at `main` branch.

Reply:
Does this mean that `run-id` is not consumed in `dev` branch at all? If so, why adding it here?

Reply:
The `run-id` is the run ID of the current job. For example, in this PR, as of now this in the URL for `Build and Verify`: https://github.com/openshift-helm-charts/repo/pull/69/checks?check_run_id=2499764781
When the job is triggered again, the `run-id` will change.

This information should be captured while this job is running. Then only we can add it as a PR comment from the other job.

Reply:
ok, so iirc, the job that consumes this `run-id` does not even exist on `dev` branch, hence this change needs to get here.